### PR TITLE
edit README.md

### PR DESCRIPTION
**Pull request by TychoSchottelius**
- first step

### CODE REVIEW

**Comment by TychoSchottelius** [comment]
Just a starting description to make the readme alive. please quickly confirm.

File: README.md [hunk 1]
```diff
@@ -1,42 +1,24 @@
 # Funding Program Manager
 
-(* FIXME: In one or two paragraphs, describe what the extension does and why one would download it. *)
+This extension forms the basis for application management in CiviCRM: manage funding programs from application to payout.
```
Comment (dontub) [inline README.md:3]: I'd suggest to use `funding application management` at this point as it is the first time the term `application` is used.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,24 @@
...
+The process of submitting and processing funding applications can be centralised:
```
Comment (dontub) [inline README.md:4]: Should be `centralized` in correspondence to the already used AE strings.

File: README.md [hunk 3]
```diff
@@ -1,42 +1,24 @@
...
+Application submission, application review, approval, call for funds, payment instruction and proof of use.
+In order to map the complex process in its entirety, this extension should be supplemented with the following:
```
Comment (dontub) [inline README.md:6]: I think we should list all extensions required in the `info.xml` (apart from the Civi core extensions) and call them `requirements` or `dependencies`. To me `should be supplemented` indicates that they are optional. Maybe we can skip them here as they will be part of the [install documentation](https://github.com/systopia/funding/blob/add-documentation/docs/installation.md)...

**Review by dontub — CHANGES_REQUESTED** [review]

**Comment by TychoSchottelius** [comment]
All requested changes done.

**Review by dontub — APPROVED** [review]

**Pull request merged**